homework/index.js

```diff
@@ -43,8 +83,12 @@
         });
         return;
       }
+      createAndAppend('h2', root, { text: 'HYF Repositories' });
       const ul = createAndAppend('ul', root);
-      repos.forEach(repo => renderRepoDetails(repo, ul));
+      const reposSorted = repos.sort(function(a, b) {
```

Comment:
The ESLint error that is reported for this line is because it prefers you use an arrow function here.